arch/lkl/include/asm/signal.h

```diff
@@ -1,5 +1,18 @@
+#pragma once
+#include <asm-generic/signal.h>
+
 struct ucontext;
+
+struct thread_info;
+struct ksignal_list_node;
+struct ksignal;
+
 void do_signal(struct pt_regs *regs);
 void lkl_process_trap(int signr, struct ucontext *uctx);
+ 
+extern void append_ksignal_node(struct thread_info *task, struct ksignal_list_node* node); 
```

Comment:
I believe Linux kernel style doesn't put `extern` on function prototypes.

Please can you add comments in the header explaining what these do?

Reply:
OK.